docs/photoniq/fingerprinting/index.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+title: Digital Fingerprinting
+---
+
+PhotonIQ Digital Fingerprinting is a managed service that generates persistent digital signatures and visitor IDs for your website users.
```

Comment:
"managed service"??? are we saying we manage the service for them?  or can an enterprise system run it without us?  Just checking on the wording "managed service".

Reply:
Updated